config/jobs/kubeflow/kubeflow-periodics.yaml

```diff
@@ -182,6 +182,38 @@ periodics:
   annotations:
     testgrid-dashboards: sig-big-data
     description: Periodic testing of Kubeflow examples
+  #******************************************************************************
+  # kubeflow/pipelines
+  #******************************************************************************
+- name: kubeflow-pipelines-periodic-functional-test
+  cluster: kubeflow
+  interval: 20m
+  decorate: true
+  labels:
+    preset-service-account: "true"
+  extra_refs:
+    - org: kubeflow
+      repo: pipelines
+      base_ref: master
+      workdir: true
+  spec:
+    containers:
+      - image: gcr.io/ml-pipeline-test/kfp-functional-test-base:latest
+        imagePullPolicy: Always
+        command:
+          - "./test/kfp-functional-test/kfp-functional-test.sh"
+        env:
+          - name: REPO_OWNER
+            value: kubeflow
+          - name: REPO_NAME
+            value: pipelines
```

Comment:
https://github.com/kubernetes/test-infra/blob/e77594eb706652194e5aa1c34cdace2ef4ad4cc0/config/jobs/kubeflow/kubeflow-periodics.yaml#L268 set the REPO_NAME to "kubeflow". I'm wondering whether this is a convention. Here I set it to "pipelines"